packages/angular_devkit/schematics/tools/file-system-engine-host_spec.ts

```diff
@@ -38,6 +38,66 @@ describe('FileSystemEngineHost', () => {
     expect(schematic1.description.name).toBe('schematic1');
   });
 
+  it('extends a collection ', () => {
```

Comment:
Should just add a test for `listSchematicNames` to avoid regression.

Reply:
done.